zebra-consensus/src/primitives/groth16.rs

```diff
@@ -217,7 +217,7 @@ impl Description for Spend<PerSpendAnchor> {
         inputs.push(rk_affine.get_u());
         inputs.push(rk_affine.get_v());
 
-        let cv_affine = jubjub::AffinePoint::from(self.cv);
+        let cv_affine = jubjub::AffinePoint::from_bytes(self.cv.0.to_bytes()).unwrap();
```

Comment:
These unwraps might look suspicious, but given that the function already uses them elsewhere, I assume they’re acceptable. Worth a second look though.